### PR TITLE
fixed up spec file

### PR DESCRIPTION
**Pull request by mslacken**
suse specifc changes for openbuild service

### CODE REVIEW

File: warewulf.spec.in [hunk 1]
```diff
@@ -30,12 +29,15 @@ Conflicts: warewulf-ipmi
 
 BuildRequires: make
 
-%if 0%{?sle_version}
+%if 0%{?suse_version}
 BuildRequires: systemd-rpm-macros
 BuildRequires: go
 BuildRequires: firewall-macros
+BuildRequires: firewalld
+BuildRequires: tftp
```
Comment (elguero) [inline warewulf.spec.in:37]: I am curious why these need to be installed in order to build the package.

File: warewulf.spec.in [hunk 2]
```diff
@@ -82,6 +84,7 @@ make
 
 
 %install
+export NO_BRP_STALE_LINK_ERROR=yes
```
Comment (elguero) [inline warewulf.spec.in:87]: Is this more specific to suse builds?

File: warewulf.spec.in (hunk 1 above)
Comment (mslacken) [reply warewulf.spec.in:37]: After the build the linting checks the ownership of all  files and dirs in the buildroot. As the makefile will install files for firewalld and tftp, these packages are needed to determine the ownership of these directories.

File: warewulf.spec.in (hunk 2 above)
Comment (mslacken) [reply warewulf.spec.in:87]: Yes, the overlay has a broken symlink for the systemd service for wwclient. The linter doesn't like it.

**Comment by mslacken** [comment]
After the build the linting checks the ownership of all  files and dirs in the buildroot. As the makefile will install files for firewalld and tftp, these packages are needed to determine the ownership of these directories.

26.04.2022 15:59:39 Michael L. Young ***@***.***>:

**Pull request merged**